app/src/main/cpp/SystemUtils.h

```diff
@@ -38,6 +38,8 @@ inline void ParseVersionString(const std::string& aString, int result[], int res
 inline bool CompareSemanticVersionStrings(const std::string& str1, const std::string& str2) {
   int parsedStr1[3]{}, parsedStr2[3]{};
```

Comment:
I'd rather initialize the array to [0, 0, 0}.

Reply:
Hmm why?. It's guaranteed that ParseVersionString will iterate over all the elements. If the parsing is correct it'll set an integer, and if it's incorrect it'll set a 0.

Reply:
I see, std::num_get ensures 0 if parsing failes. 
Thanks.